packages/volto/src/components/manage/Widgets/SchemaWidget.jsx

```diff
@@ -142,160 +171,607 @@ const makeFieldsetList = (listOfFieldsets, intl) => {
+  (factory) => {
+    config.registerUtility({
+      name: factory,
+      type: 'fieldFactoryProperties',
+      method: (intl) => ({
+        values: {
+          type: 'string',
+          title: intl.formatMessage(messages.choices),
+          widget: 'textarea',
+        },
+        default: {
+          type: 'string',
+          widget: 'textarea',
+          title: intl.formatMessage(messages.defaultValue),
+        },
+      }),
+    });
+  },
+);
+
+['Choice', 'label_choice_field', 'radio_group'].forEach((factory) => {
+  config.registerUtility({
+    name: factory,
+    type: 'fieldFactoryProperties',
+    method: (intl) => ({
+      values: {
+        type: 'string',
+        title: intl.formatMessage(messages.choices),
+        widget: 'textarea',
+      },
+      default: {
+        type: 'string',
+        title: intl.formatMessage(messages.defaultValue),
+      },
+    }),
+  });
+});
+
+config.registerUtility({
+  name: 'static_text',
+  type: 'fieldFactoryProperties',
+  method: (intl) => ({
+    default: {
+      title: intl.formatMessage(messages.text),
+      widget: 'richtext',
+      type: 'string',
+    },
+  }),
+});
+
+config.registerUtility({
+  name: 'number',
+  type: 'fieldFactoryProperties',
+  method: (intl) => ({
+    default: {
+      type: 'number',
+      title: intl.formatMessage(messages.defaultValue),
+    },
+  }),
+});
+
+config.registerUtility({
+  name: 'hidden',
+  type: 'fieldFactoryProperties',
+  method: (intl) => ({
+    default: {
+      type: 'string',
+      title: intl.formatMessage(messages.defaultValue),
+    },
+  }),
+});
+
+config.registerUtility({
+  name: 'textarea',
+  type: 'fieldFactoryProperties',
+  method: (intl) => ({
+    minLength: {
+      type: 'integer',
+      title: intl.formatMessage(messages.minLength),
+    },
+    maxLength: {
+      type: 'integer',
+      title: intl.formatMessage(messages.maxLength),
+    },
+    default: {
       type: 'string',
       widget: 'textarea',
-      title: intl.formatMessage(messages.description),
+      title: intl.formatMessage(messages.defaultValue),
     },
-    parentFieldSet: {
+    placeholder: {
       type: 'string',
-      title: intl.formatMessage(messages.parentFieldSet),
-      choices: makeFieldsetList(fieldsets),
+      title: intl.formatMessage(messages.placeholder),
     },
-    required: {
+  }),
+});
+
+// Register field factory initial data utilities
+
+['Date/Time', 'label_datetime_field'].forEach((factory) => {
+  config.registerUtility({
+    name: factory,
+    type: 'fieldFactoryInitialData',
+    method: (intl) => ({
+      type: 'string',
+      widget: 'datetime',
+      factory,
+    }),
+  });
+});
+
+['Date', 'label_date_field'].forEach((factory) => {
+  config.registerUtility({
+    name: factory,
+    type: 'fieldFactoryInitialData',
+    method: (intl) => ({
+      type: 'string',
+      widget: 'date',
+      factory,
+    }),
+  });
+});
+
+config.registerUtility({
```

Comment:
This one is missing a `['Time'...]` wrapper. Does that matter? I noticed several others lack this pattern.

Reply:
This is how it is suppose to be. The array notation is used to register multiple factories, without is just registering one.

All needed docs are added, I replied to David in this PR above.